### PR TITLE
[FIX] Provide better Dutch translations 🇳🇱

### LINKED ISSUE #12716: User_left is not translated by I18n
When a user leaves a channel, in Dutch the message 'gebruiker links' is displayed as a literal translation of 'user' (gebruiker) and 'left' (links). This 'links' translates to the direction 'left', not the verb 'to leave'. Furthermore, in the i18n-files, the translation 'Heeft de kamer verlaten' is given, which is a good translation. 

I suppose that 'User_left' is not recognized by the i18n, after which the literal translation kicks in. I suppose this bug also occurs at all other translations, as the I18n-file for Dutch is correct.

### PR DESCRIPTION
**Pull request by mathysie**
Refs #12716

Closes #12716 

This PR provides better translations in the Livechat I18n files to Dutch.

### CODE REVIEW

**Comment by sampaiodiego** [comment]
@mrsimpson can you also review this pls? thx

**Comment by mrsimpson** [comment]
Umm - no, sorry. There’s only two letters difference between Deutsch and Dutch, but there’s many more differences as you go into the actual vocabulary ;-)
I don’t speak any Dutch  🇳🇱

**Comment by sampaiodiego** [comment]
lol, sorry. yeah, I completely over looked this, sry, I swear I read Deutch :de: :disappointed:

File: packages/rocketchat-livechat/.app/i18n/nl.i18n.json [hunk 1]
```diff
@@ -25,21 +25,21 @@
...
   "Send": "sturen",
   "Skip": "Overslaan",
   "Start_Chat": "Start Chat",
   "Survey": "Enquête",
-  "Survey_instructions": "Beoordeel elke vraag naar mate uw tevredenheid, 1 betekent dat u helemaal ontevreden  bent en 5 betekent volledig tevreden.",
+  "Survey_instructions": "Beoordeel elke vraag naar mate uw tevredenheid, 1 betekent dat u helemaal ontevreden bent en 5 betekent volledig tevreden.",
   "Thank_you_for_your_feedback": "Hartelijk dank voor uw feedback",
-  "Thanks_We_ll_get_back_to_you_soon": "Bedankt! We zullen contact met u binnenkort.",
-  "transcript_sent": "Transcript verzonden",
+  "Thanks_We_ll_get_back_to_you_soon": "Bedankt! We zullen contact met u opnemen binnenkort.",
```
Comment (wampiedriessen) [inline packages/rocketchat-livechat/.app/i18n/nl.i18n.json]: -> We zullen binnenkort contact met u opnemen?

Comment (tassoevan) [reply packages/rocketchat-livechat/.app/i18n/nl.i18n.json]: @wampiedriessen Sorry, what is the issue here?

**Comment by wampiedriessen** [comment]
I suggested a better phrasing

Op vr 7 dec. 2018 20:13 schreef Tasso Evangelista <notifications@github.com:

File: packages/rocketchat-livechat/.app/i18n/nl.i18n.json (hunk 1 above)
Comment (wampiedriessen) [reply packages/rocketchat-livechat/.app/i18n/nl.i18n.json]: "Bedankt! We zullen binnenkort contact met u opnemen"
Would be a better phrasing

Comment (tassoevan) [reply packages/rocketchat-livechat/.app/i18n/nl.i18n.json]:
```suggestion
  "Thanks_We_ll_get_back_to_you_soon": "Bedankt! We zullen binnenkort contact met u opnemen.",
```

**Pull request merged**